packages/addons-inject/Dockerfile

```
@@ -3,5 +3,5 @@ COPY package.json /usr/src/app/packages/addons-inject/
 COPY index.js /usr/src/app/packages/addons-inject/
 COPY wrangler.toml /usr/src/app/packages/addons-inject/
 WORKDIR /usr/src/app/packages/addons-inject/
-RUN npm install --omit dev
-ENTRYPOINT ["node_modules/.bin/wrangler", "dev"]
+RUN npm install -g --install-links --omit dev
+ENTRYPOINT ["/usr/local/lib/node_modules/@readthedocs/addons-inject/node_modules/.bin/wrangler", "dev"]
```

Changing this to:

```suggestion
CMD ["/usr/local/lib/node_modules/@readthedocs/addons-inject/node_modules/.bin/wrangler", "dev"]
```

Makes it work fine for me.

---

I'm 99% sure we don't want ENTRYPOINT here: https://stackoverflow.com/a/21564990

---

> Makes it work fine for me.
>
> `wrangler_1     | ✘ [ERROR] Unknown argument: dev`

Both of these indicate you're probably using an old docker compose file.

There should be no `dev` command passed into the container, and replacing ENTRYPOINT should break your docker compose container execution.

Here is what you will get if you replace ENTRYPOINT with CMD, `git submodule update`, remove the `community_wrangler` image, `inv docker.build`, and `inv docker.up`:

```
wrangler_1     | node: bad option: --log-level=info
wrangler_1     | node: bad option: --host=nginx:8080
wrangler_1     | node: bad option: --ip=0.0.0.0
wrangler_1     | node: bad option: --port=8000
```

This is because the docker compose file is only passing options into the entrypoint command.

Is your `common/` up to date? And have you fully _removed_ and rebuilt your images? You do have to fully remove the image and rebuild.

Entrypoint is meant to be used when you execute containers as a single command:
https://docs.docker.com/reference/dockerfile/#:~:text=ENTRYPOINT%20should,as%20an%20executable